jcl/src/java.base/share/classes/java/lang/Access.java

```diff
@@ -714,6 +714,11 @@ public boolean isPrimaryType(Class<?> c) {
 	public boolean isPrimitiveValueType(Class<?> c) {
 		return c.isPrimitiveValueType();
 	}
+
+	@Override
+	public int classFileFormatVersion(Class<?> c) {
```

Comment:
should this be ifdef'd to java20+

Reply:
Only valhalla has package access to `Class.getClassFileVersion()` [1].
JDK 20+ `Class.getClassFileVersion()` is a private method [2].
So this can't be applied to JDK20+ yet.

[1] https://github.com/openjdk/valhalla/blob/5e871a477f79a7407053fdd4f5b4cd2aadcf8901/src/java.base/share/classes/java/lang/Class.java#L4871-L4874
[2] https://github.com/ibmruntimes/openj9-openjdk-jdk/blob/6732b1ba19e481b736e5ab43753099379b50af82/src/java.base/share/classes/java/lang/Class.java#L4691-L4694